quickwit-search/src/root.rs

```diff
@@ -269,12 +266,8 @@ pub async fn root_search(
                 };
                 let mut search_client_clone = search_client.clone();
                 let handle = tokio::spawn(async move {
-                    match search_client_clone
-                        .client()
-                        .fetch_docs(fetch_docs_request)
-                        .await
-                    {
-                        Ok(resp) => Ok(resp.into_inner()),
+                    match search_client_clone.fetch_docs(fetch_docs_request).await {
+                        Ok(resp) => Ok(resp),
                         Err(err) => Err(anyhow::anyhow!("Failed to fetch docs due to {:?}", err)),
```

Comment:
Why do we convert the error here? Wouldn't it be better to not convert the error and change the error type on line 258?

Reply:
Yes, indeed. I will change the error type.